web/modules/custom/dpl_opening_hours/src/Model/OpeningHoursRepository.php

```diff
@@ -78,6 +78,10 @@ public function loadMultiple(int $branchId = NULL, \DateTimeInterface $fromDate
       $query->condition('repetition_id', $repetitionId);
     }
 
+    if ($categoryId) {
+      $query->condition('category_tid', $categoryId);
+    }
+
```

Comment:
```suggestion
    if ($categoryId) {
      $query->condition('category_tid', $categoryId);
    }

```

This is really just nitpicking but I do not think we need a space between this check and the others which perform a similar function/purpose.